build/yaml/functionalTestPython.yml

```diff
@@ -0,0 +1,18 @@
+  steps:
```

Comment:
Rename this file as pythonBotsBuild-CI.yml

Reply:
updated with new name pythonBotsBuild-CI.yml